### PR TITLE
Fix questionnaire

### PR DESCRIPTION
**Pull request by bambolelooo**

### CODE REVIEW

**Review by DexterDick — APPROVED** [review]
looks good

File: assets/html/journey.html [hunk 1]
```diff
@@ -1,538 +1,706 @@
...
+                        >Select your goal</label
+                    >
+                    <select
+                        id="weight-goal"
+                        class="border-2 border-emerald-700 text-emerald-800 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5"
+                    >
+                        <option value="gain">Gain Muscle</option>
+                        <option value="lose">Lose Weight</option>
+                    </select>
+                </div>
```
Comment (lavanyavirushan) [inline assets/html/journey.html:477]: need the question

**Review by lavanyavirushan — CHANGES_REQUESTED** [review]
Looks good, just make sure to add question leading to the checkbox.

File: assets/html/journey.html (hunk 1 above)
Comment (bambolelooo) [reply assets/html/journey.html:477]: ok

**Review by nnavg — COMMENTED** [review]
looks fine

**Review by lavanyavirushan — APPROVED** [review]
looks good

**Pull request merged**